Doc/whatsnew/3.11.rst

```diff
@@ -715,6 +715,12 @@ Porting to Python 3.11
   been included directly, consider including ``Python.h`` instead.
   (Contributed by Victor Stinner in :issue:`35134`.)
 
+* The :c:func:`PyUnicode_CHECK_INTERNED` macro has been excluded from the
+  limited C API, because it uses the ``PyASCIIObject`` structure which is
+  excluded from the limited C API.
+  (Contributed by Victor Stinner in :issue:`46007`.)
```

Comment:
It used ``PyUnicodeObject`` in 3.2, so I'd either omit the reference to ``PyASCIIObject`` or include both.

```suggestion
* The :c:func:`PyUnicode_CHECK_INTERNED` macro has been formally excluded from the
  limited C API. It was never usable there, because it used internal structures
  which are not available in the limited C API.
  (Contributed by Victor Stinner in :issue:`46007`.)
```

Reply:
Ok, I applied your suggestion.